.gitignore

```diff
@@ -2,4 +2,6 @@ node_modules
 .history
 .assets2
 .vscode
+modal.js
+modal.html
```

Comment:
- [ ] [OPTIONAL] I noticed your `modal.js` and `modal.html` files does not have any usefulness as far as this milestone is concern and i can see you are trying to ignore them using the gitignore file. instead of that i would kindly suggest you delete the two file from your work directly so that we you can have only the files that are useful for your project and the project can be professionally documented. 👍 👍

Reply:
Thank you for the time to review my code @brainconnect93 😎

Reply:
All changes made🤗